modules/dop/endpoints/project.go

```diff
@@ -681,3 +681,158 @@ func (e *Endpoints) ParseProjectTemplate(ctx context.Context, r *http.Request, v
 	}
 	return httpserver.OkResp(templateData)
 }
+
+func (e *Endpoints) ExportProjectPackage(ctx context.Context, r *http.Request, vars map[string]string) (httpserver.Responser, error) {
+	var exportReq apistructs.ExportProjectPackageRequest
+	projectID, err := e.getProjectID(vars)
+	if err != nil {
+		return apierrors.ErrExportProjectPackage.InvalidParameter("projectID").ToResp(), nil
+	}
+	exportReq.ProjectID = projectID
+	orgID, err := e.getOrgID(vars)
+	if err != nil {
+		return apierrors.ErrExportProjectPackage.InvalidParameter("orgID").ToResp(), nil
+	}
+	exportReq.OrgID = orgID
+	// check permission
+	identityInfo, err := user.GetIdentityInfo(r)
+	if err != nil {
+		return apierrors.ErrExportProjectPackage.NotLogin().ToResp(), nil
+	}
+	exportReq.IdentityInfo = identityInfo
+	if exportReq.OrgID == 0 || exportReq.ProjectID == 0 {
+		return apierrors.ErrExportProjectPackage.InvalidParameter(fmt.Errorf("orgID and projectID can't be empty")).ToResp(), nil
+	}
```

Comment:
You can add a func to get orgID / projectID / IdentityInfo.
Duplicate code in many places